src/permission-status.ts

```diff
@@ -0,0 +1,19 @@
+import {postMessageToNativeApp} from './post-message';
+
+const TIMEOUT = 200;
+
+export const hasNotificationsPermissions = (): Promise<boolean> =>
```

Comment:
I'd keep this function generic (receive `feature` and `params` by parameters), name it for example: `checkPermissionStatus(feature: string, params: {[string]: string})`

In webapp side you can write the semantic method (`hasNotificationPermission`)

Also, don't create a new file for this, you can put this function inside utils